### PR TITLE
✨ Add Profile page

### LINKED ISSUE #14: Profile Page
### Descripition

 Site to show the main menu

##To Do:

- [ ] Profile ( img, name, status)
- [ ] build component of <ul> 
- [ ]   implement nav-bar

### PR DESCRIPTION
**Pull request by dhnazarenko**
fixes #14

### CODE REVIEW

**Review by lmachens — APPROVED** [review]
(～￣▽￣)～

**Review by tmh9x — APPROVED** [review]

File: src/app/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <ProfileMenuBar
+          link="#"
+          icon={<NotificationsIcon />}
+          text="Notifications"
+        />
+        <ProfileMenuBar link="#" icon={<SettingsIcon />} text="Settings" />
+        <ProfileMenuBar link="#" icon={<HelpIcon />} text="Help" />
+        <ProfileMenuBar link="#" icon={<LogoutIcon />} text="Logout" />
+      </main>
+      <footer className={styles.footer}>
```
Comment (jujuzitroni) [inline src/app/pages/Profile/Profile.tsx]: a NavBar component is now available and can be used here.

**Review by Dev-dfm — COMMENTED** [review]
great work! 💯

File: src/app/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+function Profile(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <Avatar imageSrc="/dieter.jpeg" />
+        <h1 className={styles.profileName}>Dieter Bohlen</h1>
+        <span className={styles.profileStatus}>Premium</span>
+      </header>
+      <main className={styles.profileMenu}>
+        <ProfileMenuBar link="#" icon={<AccountIcon />} text="Account" />
```
Comment (jujuzitroni) [inline src/app/pages/Profile/Profile.tsx:21]: maybe add a profile menu component to summarise the ProfileMenuBar components into.

File: src/app/pages/Profile/Profile.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import SearchIcon from '../../components/Icons/SearchIcon';
+import DownloadIcon from '../../components/Icons/DownloadIcon';
+import ProfileIcon from '../../components/Icons/ProfileIcon';
+
+function Profile(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <Avatar imageSrc="/dieter.jpeg" />
+        <h1 className={styles.profileName}>Dieter Bohlen</h1>
```
Comment (jujuzitroni) [inline src/app/pages/Profile/Profile.tsx:17]: h1 and span should accept dynamic input

**Review by daebert — COMMENTED** [review]
Good work!

File: src/app/pages/Profile/Profile.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+export default {
+  title: 'Page/Profile',
+  component: Profile,
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export const Default = (): JSX.Element => <Profile />;
```
Comment (jujuzitroni) [inline src/app/pages/Profile/Profile.stories.tsx:12]: can you add another story maybe with the other modern talking member? :)

**Review by SGierszewski — APPROVED** [review]
Good job!! 🥳

**Review by JGieseWeb — APPROVED** [review]

File: src/app/pages/Profile/Profile.module.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 2em;
+}
+
+.profileName {
+  margin: 0;
+  font-weight: 400;
```
Comment (lajanzen) [inline src/app/pages/Profile/Profile.module.css:18]: Isn't this the standard font-weight?

**Review by lajanzen — CHANGES_REQUESTED** [review]
Looks good to us, just a small change! 🤓

**Review by jujuzitroni — COMMENTED** [review]
good job! 
you could add two components to the Profile page, this would reduce the amount of imports you need right now.

File: src/app/pages/Profile/Profile.tsx (hunk 3 above)
Comment (dhnazarenko) [reply src/app/pages/Profile/Profile.tsx:17]: h1 and span are integrated with the api.

File: src/app/pages/Profile/Profile.tsx (hunk 2 above)
Comment (dhnazarenko) [reply src/app/pages/Profile/Profile.tsx:21]: We have discussed that this component will not be created for the time being.

File: src/app/pages/Profile/Profile.tsx (hunk 1 above)
Comment (dhnazarenko) [reply src/app/pages/Profile/Profile.tsx]: See description. Componente is built in today.

File: src/app/pages/Profile/Profile.module.css (hunk 5 above)
Comment (dhnazarenko) [reply src/app/pages/Profile/Profile.module.css:18]: I use the font weight to represent the h1 smaller. The font weight of an h1 is not defined in the global.css

File: src/app/pages/Profile/Profile.stories.tsx (hunk 4 above)
Comment (dhnazarenko) [reply src/app/pages/Profile/Profile.stories.tsx:12]: We discussed in the group that it is not necessary. Thanks for your review! 🚀

**Pull request merged**